GPSTest/src/main/res/values/do_not_translate.xml

```diff
@@ -52,6 +52,11 @@
     <string name="pref_gps_min_time_default_sec">1</string>
     <string name="pref_gps_min_distance_default_meters">0</string>
 
+    <!-- Send location URLs -->
+    <string name="google_maps_share_url">http://maps.google.com/maps?geocode=&amp;q=%1$.7f,%2$.7f\n</string>
+    <string name="openstreetmap_share_url">https://www.openstreetmap.org/?mlat=%1$.7f&amp;mlon=%2$.7f#map=18/%1$.7f/%2$.7f</string>
+    <string name="geo_share_url">http://maps.google.com/maps?geocode=&amp;q=%1$s,%2$s\n</string>
```

Comment:
Yes, geo string should be in the format of `geo:37.7749,-122.4194` - I just haven't gotten that far yet in this WIP PR :).

>Oh, and the google_maps should probably use https instead of http.

Good catch, thanks!  I added that as a TODO item in the PR too.